### ISSUE TITLE
conflicts with World Quest List

### ISSUE THREAD

**Issue by Ifnis** [NielsHeltner/wowhead-quick-link#30]
The slash command to open the config conflicts with the slash command for (the addon) World Quest List.  If both are installed, World Quest List owns the slash command and Wowhead Quick Link will not operate.

Honestly, I can't seem to get this to operate at all.  Pressing Ctrl-C while hovering over various items in my bags caused the character pane to open and close.  Changing the character pane keybind and trying Ctrl-C had no result.  Having the configuration window open did not result in the URL functioning, or getting any result.  Disabling all other addons did not change this behavior.

**Comment by veljkovicm** [comment]
Documentation doesn't mention this, but I see in the code that you can also use the `/wowheadquicklink` command.

Did you check if you have the keybinds set? For some reason I did not have them set, even though it should be done by default.

- Go to: Options > Keybindings > Addons
- Check if "Generate Wowhead link" and "Generate Raider.IO link" are bound to anything

**Comment by Ifnis** [comment]
ok, so the keybinds weren't set, with a fresh install.  However, setting the keybind doesn't actually generate a link, it opens the copy window and all it has in it is the character for the keybind I set ( [ ). (screenshot 2024-08-20 161723)

I changed the keybind to ctrl-[, and then it worked? So freaking weird.  But it worked, so I guess yay? (162259)
![Screenshot 2024-08-20 161723](https://github.com/user-attachments/assets/18ddceb5-3ecb-4171-b50d-c43814f1f83a)
![Screenshot 2024-08-20 162259](https://github.com/user-attachments/assets/4f5f5148-efe5-4691-b550-83e49013281d)

Thanks for the help, I'll go merrily on my way now.  Maybe update the user docs on CurseForge? Cheers, Ifnis
